sru/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.5.1'
+__version__ = '2.6.0.dev2'
```

Comment:
oh is this 2.x? for some reason I assumed it was 3.x. Though it does say 2.x in the title actually, now that I look ... :)

Reply:
oh, there are two PRs :P